DSL/Search.php

```diff
@@ -223,6 +223,62 @@ public function setFields(array $fields)
     }
 
     /**
+     * Set filters.
+     *
+     * @param BuilderInterface[] $filters
+     *
+     * @return $this
+     */
+    public function setFilters($filters)
```

Comment:
I don't think this is correct. Because we contain filters in bool container and it is handled by search object. This setter is also incorrect because it says that parameter `$filters` is a `BuilderInterface[]` but it should be `Bool`. Same goes to setter for post filters. For aggregations and suggesters it's a no also. We intentionally wrapped them in Search object because they use same container type and by not exposing them could have some changes internally, which user would not even notice.